### PR TITLE
Merge the capabilities, authorization vocabularies from OASIS TC github

### PR DESCRIPTION
**Pull request by xuzhg**
…ub repo

### Issues

* Merge the capabilities, authorization vocabularies from OASIS TC

### Description

* Merge the capabilities, authorization vocabularies from OASIS TC

### Checklist (Uncheck if it is not completed)

- [x] *Test cases added*
- [x] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml [hunk 1]
```diff
@@ -593,6 +609,36 @@ supported:
...
+          <Annotation Term="Core.LongDescription">
+            <String>Possible string value identifiers when specifying properties are `*`, _PropertyName_, `-`_PropertyName_.
+
+`*` denotes all properties are accessible.
+
+`-`_PropertyName_ excludes that specific property.
+
+_PropertyName_ explicitly provides access to the specific property.
+
+The absence of `RestrictedProperties` denotes all properties are accessible using that scope.</String>
```
Comment (mikepizzo) [inline src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml]: Can we clean up the formatting a bit?

File: src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml [hunk 2]
```diff
@@ -744,13 +812,14 @@ supported:
 
       <Term Name="ReadRestrictions" Type="Capabilities.ReadRestrictionsType" AppliesTo="EntitySet Singleton FunctionImport">
         <Annotation Term="Core.Description"
-          String="Restrictions on read operations: retrieve a collection, retrieve a single instance, invoke a function" />
+          String="Restrictions for retrieving a collection of entities, retrieving a singleton instance, invoking a function" />
       </Term>
-      <ComplexType Name="ReadRestrictionsType">
```
Comment (mikepizzo) [inline src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml]: Did we lose ReadRestrictionsType?   We should have:

ReadRestrictionsBase
ReadRestrictionsType : ReadRestrictionsBase
ReadRestrictionsByKey : ReadRestrictionsBase

Comment (xuzhg) [reply src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml]: No. See line 833. We have both.

**Comment by madansr7** [comment]
Looks good!

File: src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml (hunk 2 above)
Comment (mikepizzo) [reply src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml]: Got it; missed that with all of the highlighting. Thanks.

---
In reply to: [268340091](https://github.com/OData/odata.net/pull/1428#discussion_r268340091) [](ancestors = 268340091)

**Pull request merged**